src/store/conversationsStore.js

```diff
@@ -439,6 +440,20 @@ const actions = {
 		commit('addConversation', conversation)
 	},
 
+	async markConversationUnread({ dispatch, getters }, { token }) {
+		const conversation = Object.assign({}, getters.conversations[token])
+		if (!conversation) {
+			return
+		}
+
+		try {
+			await setConversationUnread(token)
+			dispatch('fetchConversation', { token })
```

Comment:
Instead of dispatching a fetch, can we directly manipulate the data?
Unread counter is set to 1?

… Ah, unreadMention could be a problem?

Reply:
would still prefer to see it bold + counter immediate, can turn blue after another server round trip (or we fix the api)

Reply:
```suggestion
      conversation.unreadMessages = 1
			dispatch('fetchConversation', { token })
```

Could be patched like this, so if `axios.delete` will return successful response, we could see that immediately, and fetch and update conversation afterwards (because on server it also updates `lastReadMessage` and should be received)